package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ushahidi-platform-pattern-library",
   "description": "Ushahidi Platform Pattern Library",
-  "version": "3.5.0",
+  "version": "3.5.1",
```

Comment:
this was published before your documentation was written. I will reach out if needed.